src/components/WelcomeIntro.js

```diff
@@ -35,24 +113,16 @@ const WelcomeIntro = () => {
             <br />
             <br />
           </p>
-          <div>
+          <div className="slack">
+            <p className="slack__title">SLACK WORKSPACE</p>
             <a
-              className="btn primary-cta mr-3 mt-2"
+              className="slack__link"
               // TODO: Change this link to the actual slack link when it's available
-              href="#coming-soon"
-              target="_blank"
-              rel="noreferrer"
-            >
-              Join our Slack
-            </a>
-            <a
-              className="btn primary-cta mt-2"
-              // TODO: Check that this is the correct link once gathertown is set up
-              href="https://gather.town/hackbeanpot2021"
+              href={"https://hackbeanpot.com"}
               target="_blank"
               rel="noreferrer"
             >
-              Visit the Hacker Lounge
+              slackbeanpot2021.slack.com
```

Comment:
Do we want to change the slack link to a button in this PR? I think the figma uses two buttons to link instead of text here

Reply:
ah whoops sorry I think in main it is a button. But not sure if this change is from before? It looks like the text should be Visit the Hacker Lounge